WordPress/src/main/java/org/wordpress/android/ui/mysite/QuickStartMenuViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package org.wordpress.android.ui.mysite
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import org.wordpress.android.ui.mysite.QuickStartMenuViewModel.QuickStartMenuInteraction.Hide
+import org.wordpress.android.ui.mysite.QuickStartMenuViewModel.QuickStartMenuInteraction.Pin
+import org.wordpress.android.ui.mysite.QuickStartMenuViewModel.QuickStartMenuInteraction.Remove
+import org.wordpress.android.viewmodel.Event
+import javax.inject.Inject
+
+class QuickStartMenuViewModel
```

Comment:
Could you please add a test for this class? Even if they are quite dumb 👍

Reply:
Will do! 👍